package.json

```diff
@@ -102,14 +102,14 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-helmet-async": "^1.3.0",
+    "react-helmet-async": "^2.0.4",
     "react-helsinki-headless-cms": "1.0.0-alpha253",
```

Comment:
If you're upgrading hds-react to v3.7.0 I think it'd be good to upgrade the react-helsinki-headless-cms to the newest one too as then they'd be using the same version of hds-react.

Reply:
Good call. Upgraded